tools/travis/install_script.sh

```diff
@@ -23,14 +24,18 @@ if [[ $SUITE == "client" ]]; then
     pip install -r tools/pip-requires-test
     python setup_rucio_client.py install
     cp etc/docker/travis/rucio_client.cfg etc/rucio.cfg
-
     cp etc/docker/travis/Dockerfile Dockerfile
     docker build -t rucio/rucio .
 
 elif [[ $SUITE == "syntax" ]]; then
     pip install setuptools_scm
-    pip install  google_compute_engine
+    pip install google_compute_engine
```

Comment:
If needed by Rucio, it should be included in the dependencies.  If not necessary, it can be be put `extra` dependencies

Reply:
i didn't put it there, don't even know what this is about. i just removed the extra whitespace ;-) should be fixed by the one who put it there.